cdap-master/src/main/java/co/cask/cdap/data/tools/Main.java

```diff
@@ -71,6 +73,7 @@
    */
   private enum Action {
     UPGRADE("Upgrade all tables."),
+    MIGRATE_METRICS_DATA("Migrate metrics data to CDAP-2.8"),
```

Comment:
"Migrate metrics data".

Also, we want to say why this is a separate action in a comment: "Metrics migration is not required for CDAP to work after upgrade, some may opt to not do it as it may take a while"